.github/workflows/ci.yml

```diff
@@ -304,7 +304,44 @@ jobs:
       - name: "Clean docker cache for ${{ matrix.platform }}"
         run: docker system prune --all --force
         if: matrix.platform == 'linux/amd64' && needs.build-info.outputs.merge-run == 'true'
-      - name: "Check that image builds quickly ${{ matrix.platform }}"
+      - name: "Fix ownership"
+        run: breeze ci fix-ownership
+        if: always() && needs.build-info.outputs.merge-run == 'true'
+
+  # Check that after earlier cache push, breeze command will build quickly
+  chcek-that-image-builds-quicklly:
+    timeout-minutes: 50
```

Comment:
50mins is waaay longer than we ever expect it to take, right?

Reply:
It's a copy paste victim. But it does not matter. We have the "--max-time" parameter in Breeze that sets a timer that will actually fail after 2 minutes anyway. I will fix it :)